taskmanager/features/task.feature

```diff
@@ -9,3 +9,32 @@
       When I create a task named "Add switch language button"
       Then there should be "1" tasks with "TODO" status
 
+    Scenario: Change task status from TODO to IN PROGRESS
+      Given there is a task named "Add switch language button" with "TODO" status
+      When I move this task to "IN PROGRESS" column
+      Then there still will be "1" task
+      And it should have status "IN PROGRESS"
+
+    Scenario: Change task status from IN PROGRESS to DONE
+      Given there is a task named "Add switch language button" with "IN PROGRESS" status
+      When I move this task to "DONE" column
+      Then there still will be "1" task
+      And it should have status "DONE"
+
+    Scenario: Change task status from DONE to CLOSED
+      Given there is a task named "Add switch language button" with "DONE" status
+      When I move this task to "CLOSED" column
+      Then there still will be "1" task
+      And it should have status "CLOSED"
+
+    Scenario: Change task status from IN PROGRESS to TODO
+      Given there is a task named "Add switch language button" with "IN PROGRESS" status
+      When I move this task to "TODO" column
+      Then there still will be "1" task
+      And it should have status "TODO"
+
+    Scenario: Change task status from DONE to IN PROGRESS
+      Given there is a task named "Add switch language button" with "DONE" status
+      When I move this task to "IN PROGRESS" column
+      Then there still will be "1" task
+      And it should have status "IN PROGRESS"
```

Comment:
As extra story please add story from CLOSE to some other status (e.g. TODO) and throw exception then. As we do not allow open closed tasks.

Reply:
```gherkin
Scenario: Change task status from CLOSED to IN PROGRESS
    Given there is a task named "Add switch language button" with "CLOSED" status
    When I change task "Add switch language button" status to "IN PROGRESS"
    Then task "Add switch language button" should have status "CLOSED"
```

Reply:
Added